resources/assets/js/app/views/topics/topicItem.js

```diff
@@ -1,7 +1,13 @@
 var Marionette = require('backbone.marionette');
+var moment = require('momentjs');
 
 module.exports = Marionette.ItemView.extend({
     template: 'topicItem',
     className: 'row post-item',
-    tagName: 'div'
+    tagName: 'div',
+    
+    initialize: function () {
+        this.model.attributes.created_at =
+            moment(this.model.attributes.created_at).format('dd.MM.YYYY');
```

Comment:
Ok, I agree. I included this fix in my previous pull request, to prevent conflicts.